kymatio/scattering1d/core/timefrequency_scattering.py

```diff
@@ -13,7 +13,7 @@ def joint_timefrequency_scattering(U_0, backend, filters, oversampling,
         Yields scattering coefficients at the sample
         rate max(1, 2**(log2_T-oversampling)). Hence, raising oversampling by
         doubles the sample rate, until reaching the native sample rate.
-    average_local : boolean, optional
+    average_local : boolean
```

Comment:
not optional any more? always need to be specified by the user whether to average?

Reply:
it's optional in the frontend, it's compulsory in core

Reply:
I see.